lib/services/networkManagement2/lib/models/index.d.ts

```diff
@@ -2484,7 +2544,6 @@ export interface RouteFilterRule extends SubResource {
   name?: string;
   location?: string;
   readonly etag?: string;
-  tags?: { [propertyName: string]: string };
```

Comment:
Removing an interface property is a breaking change. Requires a major version increment.